serializations.md

```diff
@@ -644,6 +646,22 @@ The payload is a serialized signed channel off-chain transaction and can not be
 ]
 ```
 
+version 2:
+```
+[ <channel_id>      :: id()
+, <from_id>         :: id()
+, <payload>         :: binary()
+, <round>           :: int()
+, <update>          :: binary()
+, <state_hash>      :: binary()
+, <offchain_trees>  :: trees()
+, <ttl>             :: int()
+, <fee>             :: int()
+, <nonce>           :: int()
+, <block_hash>       :: binary()
+]
+```
+
 The payload is a serialized co-signed channel off-chain transaction or it is empty.
```

Comment:
From this description it looks like channel solo force progress transaction version 1 can include channel off-chain transaction version 2. (That would be a backward incompatible change in current Minerva protocol, hence a bug.)

Reply:
version2 serializations will be valid since Fortuna hardfork, but version1 will not be invalid after it. After Fortuna hardfork a version1 Force Progress will be valid if including a version 2 off-chain transacttion it was based upon as a payload